pages/Getting Started/Installation.md

```diff
@@ -165,9 +165,12 @@ Hyprland and related are in the default repository:
 
 {{% /details %}}
 
-{{% details title="Ubuntu 23.04*" closed="true" %}}
-
-Build Dependencies:
+{{% details title="Ubuntu*" closed="true" %}}
+Hyprland made it into the universe repos and can be installed with 
+```bash
+sudo add-apt-repository universe && sudo apt-get update && sudo apt-get install -y hyprland
+```
+Build dependencies:
```

Comment:
Can you add a newline after the `{{% details` line and before and after the codeblock?